.vscodeignore

```diff
@@ -1,17 +1,45 @@
-.vscode/**
-.vscode-test/**
-out/test/**
-out/**/*.map
-out/**/*.ts
-src/**
+.vscode
+.vscode-test
+!node_modules/keytar
+!node_modules/getmac
+!node_modules/editions
+!node_modules/errlop
+!node_modules/extract-opts
+!node_modules/typechecker
+!node_modules/eachr
+!node_modules/getos
+!node_modules/node-usb-native/native
+!node_modules/usb-detection/build/Release/detection.node
+node_modules/usb-detection/doc
+node_modules/usb-detection/test
+!node_modules/usb-detection/*.*
```

Comment:
Is it valid to use "!node_modules/usb-detection" instead of "!node_modules/usb-detection/*.*" here?

Reply:
There are a sub-dir in usb-detection contains some cpp files, which I don't wanna pack in, but I need to pack the source files under usb-detection. So wrote like this, but I think probably i can remove the previous 2 lines. I will change it.